server/views/pages/comDetails.njk

```diff
@@ -25,32 +25,32 @@
                             text: "Name"
                         },
                         value: {
-                            text: licence.comFirstName + " " + licence.comLastName
+                            text: name
                         }
                     },
                     {
                         key: {
                             text: "Office phone number"
                         },
                         value: {
-                            text: licence.comTelephone
+                            text: telephone
                         }
                     },
                     {
                         key: {
                             text: "Email"
                         },
                         value: {
-                            text: licence.comEmail
+                            text: email
                         }
                     }
                 ]
             }) }}
 
             {{ govukButton({
                 text: 'Return to case list',
-                href: '/licence/view/cases',
-                classes: 'govuk-!-margin-top-6',
+                href: returnLink,
+                classes: 'govuk-!-margin-top-6 js-backlink',
```

Comment:
Using the `js-backlink` class, so that the correct tab will be selected when returning to caseload